lib/cdk/app.ts

```diff
@@ -30,8 +34,11 @@ new NextStandaloneStack(app, process.env.STACK_NAME, {
 
 	lambdaTimeout: process.env.LAMBDA_TIMEOUT ? Number(process.env.LAMBDA_TIMEOUT) : 15,
```

Comment:
I'm thinking that same should be done for server-lambda as well, in terms of defining constant, I will put a note for myself

Reply:
I guess so… But I tried to minimize the diff for now.

(I wish I could say the same for my imaginex PR :stuck_out_tongue_closed_eyes: )